vue.config.js

```diff
@@ -28,7 +28,13 @@ module.exports = {
       ],
       builderOptions: {
         productName: appName,
-        appId: 'com.electron.' + appName,
+        appId: `com.electron.${appName}`,
+        publish: [
+          {
+            provider: 'github',
+            releaseType: 'release'
```

Comment:
The default is `draft`, but the release is already created by `semantic-release`, so the type is `release`, and it adding the artifact to the existing release.